recipes/mpi4py-fft/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set build = 0 %}
+{% set name = "mpi4py-fft" %}
+{% set version = "1.0.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://bitbucket.org/mpi4py/mpi4py-fft/get/{{ version }}.tar.bz2
+  sha256: d697e836d436c12da721b4bcc984f3cc99fff6e72b4a9e3a31da93bc7a89db40
+
+build:
+  number: {{ build }}
+  skip: True  # [win]
+  string: py{{ py }}_{{ mpi }}_{{ build }}
```

Comment:
I'm not following the development on how variants works for `mpi`. Maybe @isuruf and @minrk could review this one and give the :+1:

Reply:
With cb3, we no longer need to specify build strings, the variant-hashing takes care of this.

Reply:
Should I remove {{ mpi }} or do we have to wait for cb3? (What's cb3?)

Reply:
Oops, sorry! Conda-Build 3. We are using this throughout now, so you can just drop the build string entirely and everything should be okay.